### ISSUE TITLE
Overlay pinned map visualizations

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#191]
## User story
1. As a User 
2. I want to overlay different views (pinned maps)
3. So that I can see various information about an area / coordinate

## Acceptance criteria
* Integrate pinned map visualization

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
we can pin a map and see the markers in the other maps
